docs/nightly/en/user-guide/continuous-aggregation/expression.md

```diff
@@ -1,9 +1,101 @@
 # Expression
 
-This part list all supported aggregate functions.
+## aggregate functions
+
+This part list all supported aggregate functions in flow.
 
 - `count(column)`: count the number of rows.
 - `sum(column)`: sum the values of the column.
 - `avg(column)`: calculate the average value of the column.
 - `min(column)`: find the minimum value of the column.
 - `max(column)`: find the maximum value of the column.
+
```

Comment:
Do we support DataFusion aggregation functions?

Reply:
> Do we support DataFusion aggregation functions?

Not yet, but there is only 15 of them: https://datafusion.apache.org/user-guide/sql/aggregate_functions.html